src/current/v23.2/read-committed.md

```diff
@@ -80,7 +80,7 @@ To set all future transactions to run at `READ COMMITTED` isolation, use one of
 
 	{% include_cached copy-clipboard.html %}
 	~~~ sql
-	cockroach sql -–url='postgresql://{username}@{host}:{port}/{database}?options=-c default_transaction_isolation=read%20committed'
+	cockroach sql --url='postgresql://{username}@{host}:{port}/{database}?default_transaction_isolation=read committed'
```

Comment:
although the `options` parameter is more complex, unfortuntately PGJDBC, which is one of the most popular drivers, doesn't support configuring session variables in the URL unless they are in the `options` section.

i think it makes sense to keep the old example, but just fix the spacing. so it should be
```
postgresql://{username}@{host}:{port}/{database}?options=-c default_transaction_isolation=read\ committed
```

i left this comment in the thread where @smcvey initially filed this issue too: https://cockroachlabs.slack.com/archives/C04E47S4KK9/p1726758096559559?thread_ts=1726751860.181139&cid=C04E47S4KK9